packages/kit/src/views/Swap/pages/components/SwapInputActions.tsx

```
@@ -38,56 +38,70 @@ const SwapInputActions = ({
+            }}
+            gap="$0.5"
+          >
+            <>
+              {needSwapPercentageInputStage.map((stage) => (
+                <SwapPercentageStageBadge
+                  key={`swap-percentage-input-stage-${stage}`}
+                  stage={stage}
+                  onSelectStage={onSelectStage}
+                />
+              ))}
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)